src/app/user/profile/profile.page.ts

```diff
@@ -27,11 +34,11 @@ export class ProfilePage implements OnInit {
       return;
     }
 
-    const { email, password, firstName, lastName } = userEditForm.value;
+    const { email, password, firstName, lastName, organizatorName, phone} = userEditForm.value;
 
     const region = this.region;
   }
-  constructor(private authService: AuthService) {}
+  constructor(private authService: AuthService, private router: Router) {}
 
   ngOnInit() {
     if(this.authService.getUser()){
```

Comment:
I still don’t have an answer - why calling the function twice if true?

Reply:
It was late, now I did it better.